code/modules/organs/blood.dm

```diff
@@ -128,16 +128,10 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 		for(var/obj/item/organ/external/temp in organs)
 			if(!(temp.status & ORGAN_BLEEDING) || temp.status & ORGAN_ROBOT)
 				continue
-			if(isvaurca(src))
-				for(var/datum/wound/W in temp.wounds) if(W.bleeding())
-					blood_max += W.damage / 30
-				if (temp.open)
-					blood_max += 3  //Yer stomach is cut open
-			else
-				for(var/datum/wound/W in temp.wounds) if(W.bleeding())
-					blood_max += W.damage / 40
-				if (temp.open)
-					blood_max += 2  //Yer stomach is cut open
+			for(var/datum/wound/W in temp.wounds) if(W.bleeding())
+				blood_max += ((W.damage / 40) * species.fall_mod)
+			if (temp.open)
+				blood_max += 2 * species.bleed_mod  //Yer stomach is cut open
```

Comment:
This is merely a 300% increase from default values, as it is only 2x greater than old Vaurca values. This and the value above seem ridiculous to me, to the point where even minor blood-related injuries will be serious and certain things like shrapnel and other sources of internal bleeding may very well be insta-kills.